driver-legacy/src/main/com/mongodb/DB.java

```diff
@@ -105,7 +106,9 @@ public DB(final Mongo mongo, final String name) {
      * Gets the Mongo instance
      *
      * @return the mongo instance that this database was created from.
+     * @deprecated
      */
+    @Deprecated
```

Comment:
We could add a getMongoClient method to replace this.

Reply:
I think thats a good idea for those staying on the legacy API.

Reply:
Done